clap-utils/src/keypair.rs

```diff
@@ -617,6 +628,20 @@ pub struct SignerFromPathConfig {
 ///   - `usb://ledger/9rPVSygg3brqghvdZ6wsL2i5YNQTGhXGdJzF65YxaCQd`
 ///   - `usb://ledger/9rPVSygg3brqghvdZ6wsL2i5YNQTGhXGdJzF65YxaCQd?key=0/0`
 ///
+/// - `pgpcard:` &mdash; Use a connected OpenPGP smart card as the signer.
+///   In particular, use the key in the signing slot of the OpenPGP application
+///   on the card.
+///
+///   The URI host is the 16-byte OpenPGP AID in its standard hexadecimal form
+///   (32 digits). (For details, see section 4.2.1 of the
+///   [OpenPGP specification][pgpspec].) The URI host may also be omitted, in
+///   which case the first connected smart card that is found is used.
+///
+///   Examples:
+///
+///   - `pgpcard://`
+///   - `pgpcard://D2760001240103040006123456780000`
+///
```

Comment:
The main reason for the separate scheme is that PGP cards need not necessarily be connected via USB. For example, the smart card connection on YubiKeys that provide access to the OpenPGP app is also available via the Lightning transport on 5Ci models and via NFC on other models.

Such connections would, in theory, still be listed and provided by the system's underlying PCSC framework and should be "valid PGP cards". I agree that the most common use case by far would be YubiKeys (or other card readers) connected via USB, but I think having a dedicated `pgpcard` scheme makes sense in case the code is used in future projects that will leverage those other transport types.